.env.example

```diff
@@ -44,8 +44,18 @@ DB_SCHEMA=public
 
 # Prisma database connection
 
-DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${DB_HOST}:${DB_PORT}/${POSTGRES_DB}?schema=${DB_SCHEMA}&sslmode=prefer&connection_limit=400&pool_timeout=30
+DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${DB_HOST}:${DB_PORT}/${POSTGRES_DB}?schema=${DB_SCHEMA} &
```

Comment:
This string seems to be an unnecessary modification by prettier

Reply:
Oh, sorry.